hubble/relay.go

```diff
@@ -409,3 +410,24 @@ func (k *K8sHubble) createRelayClientCertificate(ctx context.Context) error {
 
 	return nil
 }
+
+func (k *K8sHubble) PortForwardCommand(ctx context.Context) error {
+	cmd := "kubectl"
+	args := []string{
+		"port-forward",
+		"-n", k.params.Namespace,
+		"svc/hubble-relay",
+		"--address", "0.0.0.0",
+		"--address", "::",
+		fmt.Sprintf("%d:80", k.params.PortForward)}
+
+	c := exec.Command(cmd, args...)
+	c.Stdout = k.params.Writer
+	c.Stderr = k.params.Writer
+
+	if err := c.Run(); err != nil {
```

Comment:
I don't quite recollect if SIGINT will get passed to the child, or do we need to do something about it?

Reply:
Interrupting it works fine so not sure we need to do something